docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+cronitor:
+  build: .
+  container_name: cronitor
+  ports:
+    - 8434:8434
+  volumes:
+    - .:/opt/cronitor
+    - ./cronitor-log:/var/log/cronitor
+  environment:
+    - VIRTUAL_HOST=cronitor.samtranet.de
```

Comment:
What is this for? Cronitor doesn't look at this environment variable at all AFAIK.